stack/server.js

```diff
@@ -3,11 +3,14 @@ const constants = require('./constants');
 const express = require('express');
 const readline = require('readline');
 const compression = require('compression');
+const slow = require('connect-slow');
```

Comment:
@LucaMele 
We could either remove `connect-slow` or keep it for future testing, but disable it for prod?

Reply:
I would comment it out..